renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":disableRateLimiting",
+    ":maintainLockFilesWeekly",
+    ":semanticCommits"
+  ],
+  "postUpdateOptions": ["pnpmDedupe"],
```

Comment:
Run dedupe every time. Combined with lock file maintenance, reproducibility and stability can be increased.
Ref: https://pnpm.io/cli/dedupe

Reply:
Soo goooood!